.gitignore

```
@@ -0,0 +1 @@
+node_modules/
```

```suggestion
node_modules/
dist/
```

빌드한 파일도 커밋되지 않도록 추가해두면 좋을 것 같아요

---

반영했습니다!